components/button/button-toggle.js

```diff
@@ -7,6 +8,11 @@ class ButtonToggle extends LitElement {
 
 	static get properties() {
 		return {
+			/**
+			 * When true, consumer is expected to manage pressed state
+			 * @type {boolean}
+			 */
+			consumerManageState: { type: Boolean, attribute: 'consumer-manage-state' },
```

Comment:
Alternate idea: could we leverage the fact that events can be cancellable? We could suppress the built-in `click` event and fire our own `cancellable` click event. The consumer would then call `preventDefault()` on it if they want to handle the behaviour themselves, and our code can check `e.defaultPrevented` to determine whether to proceed with manipulating the state.

Reply:
I think I prefer the event approach. 😄  Seems more inline with how native elements would work.